core/templates/pages/contributor-dashboard-page/custom-snackbar/undo-snackbar.component.spec.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
```

Comment:
Update headers to match current year. Please update any other phrases accordingly. Ditto elsewhere.

Reply:
Done, Thanks! -- Btw should we change names for files which I haven't created but modified ?, e.g translation-suggestion-review-modal.component.ts